### PR TITLE
feat: machines can be deployed by using more background services

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: lib/online_kitchen/labmanager4.rb [hunk 1]
```diff
@@ -124,12 +132,47 @@ def equip_machine_ip(name)
         false
       end
 
+      def get_config(opts = {})
+        # if opts.key?(:cluster)
+        #   OnlineKitchen.logger.debug("\e[1;31m LMNAME PRESENT #{opts[:cluster]}  \e[0m")
+        # else
+        #   OnlineKitchen.logger.debug("\e[1;31m LMNAME NOT PRESENT  \e[0m")
+        # end
```
Comment (petrSchreiber) [inline lib/online_kitchen/labmanager4.rb:140]: Please consider removing this unused block of code, thank you

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
Comment left in the code

File: lib/online_kitchen/labmanager4.rb (hunk 1 above)
Comment (ireznice) [reply lib/online_kitchen/labmanager4.rb]: done, even more lines removed

**Review by petrSchreiber — APPROVED** [review]
Thank you for the improvements, this opens new possibilities 🎉

**Pull request merged**